controllers/istiorevision/istiorevision_controller.go

```
@@ -133,26 +126,44 @@ func (r *IstioRevisionReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
-	values, err := applyOverridesToIstioRevision(&rev, rev.Spec.GetValues())
-	if err != nil {
-		err = r.updateStatus(ctx, logger, &rev, values, err)
+	values := rev.Spec.GetValues()
+	if err := validateIstioRevision(rev, values); err != nil {
 		return ctrl.Result{}, err
 	}
 
 	logger.Info("Installing components", "values", values)
-	err = r.installHelmCharts(ctx, &rev, values)
+	err := r.installHelmCharts(ctx, &rev, values)
```

are we ignoring errors here?

---

no, we pass `err` to `updateStatus`, which reports the error via the Istio resource and also returns it, so that its also returned from Reconcile().